examples/advanced_training/transfer_learning.py

```diff
@@ -0,0 +1,399 @@
+n_classes = 2
+input_window_samples = 6000
+drop_prob = 0.5
+cuda = True  # Set to False if you don't have a GPU
+n_start_chans = 25
+final_conv_length = 1
+n_chan_factor = 2
+stride_before_pool = True
+# input_window_samples =6000
+model = Deep4Net(
+    n_chans,
+    n_classes,
+    n_filters_time=n_start_chans,
+    n_filters_spat=n_start_chans,
+    input_window_samples=input_window_samples,
+    n_filters_2=int(n_start_chans * n_chan_factor),
+    n_filters_3=int(n_start_chans * (n_chan_factor**2.0)),
+    n_filters_4=int(n_start_chans * (n_chan_factor**3.0)),
+    final_conv_length=final_conv_length,
+    stride_before_pool=stride_before_pool,
+    drop_prob=drop_prob,
+)
+# Send model to GPU
+if cuda:
+    model.cuda()
+
+from braindecode.models.util import to_dense_prediction_model, get_output_shape
+
+to_dense_prediction_model(model)
+
+n_preds_per_input = get_output_shape(model, n_chans, input_window_samples)[2]
+
+######################################################################
+# Extracting windows
+# -------------------------------------
+#
+# We extract 60-s windows to be used in both datasets. We use a window size of 6000 samples.
+#
+
+from braindecode.datautil.windowers import create_fixed_length_windows
+
+window_train_set = create_fixed_length_windows(
+    train_set,
+    start_offset_samples=0,
+    stop_offset_samples=None,
+    preload=True,
+    window_size_samples=input_window_samples,
+    window_stride_samples=n_preds_per_input,
+    drop_last_window=True,
+)
+
+window_test_set = create_fixed_length_windows(
+    test_set,
+    start_offset_samples=0,
+    stop_offset_samples=None,
+    preload=False,
+    window_size_samples=input_window_samples,
+    window_stride_samples=n_preds_per_input,
+    drop_last_window=False,
+)
+
+######################################################################
+# Defining the classifier windows
+# -------------------------------------
+
+
+from braindecode import EEGClassifier
+from braindecode.training.losses import CroppedLoss
+import torch
+from skorch.callbacks import LRScheduler
+from skorch.helper import predefined_split
+
+weight_decay = 0.5 * 0.001
+lr = 0.0625 * 0.01
+n_epochs = 10
+batch_size = 64
+
+clf = EEGClassifier(
+    model,
+    cropped=True,
+    classes=[0, 1],
+    criterion=CroppedLoss,
+    criterion__loss_function=torch.nn.functional.nll_loss,
+    optimizer=torch.optim.AdamW,
+    train_split=predefined_split(window_test_set),
+    optimizer__lr=lr,
+    optimizer__weight_decay=weight_decay,
+    iterator_train__shuffle=True,
+    batch_size=batch_size,
+    callbacks=[
+        "accuracy",
+        "balanced_accuracy",
+        "f1",
+        ("lr_scheduler", LRScheduler("CosineAnnealingLR", T_max=n_epochs - 1)),
+    ],
+    device="cuda" if cuda else "cpu",
+)
+
+clf.initialize()  # This is important!
+print("classifier initialized")
+print(
+    "Number of parameters = ",
+    sum(p.numel() for p in model.parameters() if p.requires_grad),
+)
```

Comment:
```suggestion
print(clf)
```
This is will print the parameters because of EEGClassifier